experimental/framework/data-objects/src/kvpair/DataObject.ts

```diff
@@ -42,12 +48,33 @@ export class KeyValueDataObject extends DataObject implements IKeyValueDataObjec
     }
 
     public set = (key: string, value: any) => {
-        this.root.set(key, value);
+        this.root.set(key.toString(), value);
     };
 
     public get = (key: string) => {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return this.root.get(key);
+        return this.root.get(key.toString());
```

Comment:
Why is the toString needed here?

Reply:
ensuring the value is a string

Reply:
trying to recall why it was necessary. let me double check

Reply:
hmm....not sure why it got added. seems fine without. i'll remove